test/functional/feature_asmap.py

```
@@ -111,6 +111,14 @@ def test_empty_asmap(self):
         self.node.assert_start_raises_init_error(extra_args=['-asmap'], expected_msg=msg)
         os.remove(self.default_asmap)
 
+    def test_asmap_health_check(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_asmap_health_check`

Add return type annotation: `None`

(ANN201)